homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -57,13 +57,24 @@ async def async_setup_entry(
     pv_entry: PowerviewEntryData = hass.data[DOMAIN][entry.entry_id]
     coordinator: PowerviewShadeUpdateCoordinator = pv_entry.coordinator
 
+    async def _async_initial_refresh() -> None:
+        """Force position refresh shortly after adding.
+
+        Legacy shades can become out of sync with hub when moved
+        using physical remotes. This also allows reducing speed
+        of calls to older generation hubs in an effort to
+        prevent hub crashes.
+        """
+
+        for shade in pv_entry.shade_data.values():
+            with suppress(asyncio.TimeoutError):
```

Comment:
```suggestion
            with suppress(TimeoutError):
```

https://docs.python.org/3/library/asyncio-exceptions.html#asyncio.TimeoutError